### ISSUE TITLE
Bug in 005.js - Network name not passed to getMe() function

### ISSUE THREAD

**Issue by tomhatzer** [caffeinery/coffea#75]
Hi!

On line 19 of file lib/plugins/005.js you missed to pass the network to the function call, causing named networks to close the connection right after connecting.

This:
```javascript
var data = event.params.replace(irc.getMe().nick, '');
```

Should be this:
```javascript
var data = event.params.replace(irc.getMe(event.network).nick, '');
```

The bug causes the following fatal error:
```
coffea/lib/plugins/nick.js:27
            res.client = null;
                       ^

TypeError: Cannot set property 'client' of undefined
    at Client.ircGetMe [as getMe] (coffea/lib/plugins/nick.js:27:24)
    at coffea/lib/plugins/005.js:19:49
    at Client.func (coffea/index.js:378:13)
    at Client.emit (coffea/node_modules/eventemitter3/index.js:90:35)
    at Object.exports.emit (coffea/lib/utils.js:73:25)
    at Client.<anonymous> (coffea/lib/plugins/numeric.js:150:23)
    at Client.emit (coffea/node_modules/eventemitter3/index.js:111:33)
    at Object.exports.emit (coffea/lib/utils.js:73:25)
    at Client.onmessage (coffea/index.js:205:11)
    at Parser.<anonymous> (coffea/index.js:164:15)
```

**Comment by omnidan** [comment]
Thanks a lot for finding (and even debugging!) this issue! :grin:

I'll fix it and release a new version

**Comment by omnidan** [comment]
fixed in `0.4.22`
